MainWindow.xaml

```diff
@@ -4,9 +4,11 @@
         xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:JiME"
+        xmlns:localviews="clr-namespace:JiME.Views"
+        xmlns:controls="http://schemas.panthernet.ru/graphx/"
         mc:Ignorable="d"
 				xmlns:uc="clr-namespace:JiME.UserControls"
-        Title="{Binding titleChangedToken, Converter={StaticResource titleConverter}}" Height="500" Width="770" MinHeight="500" MinWidth="700" WindowStartupLocation="CenterScreen" Background="#FF2D2D30" Closing="Window_Closing" ResizeMode="CanResizeWithGrip" SizeToContent="Width">
+        Title="{Binding titleChangedToken, Converter={StaticResource titleConverter}}" Height="500" Width="770" MinHeight="500" MinWidth="700" WindowStartupLocation="CenterScreen" Background="#FF2D2D30" Closing="Window_Closing" ResizeMode="CanResizeWithGrip" SizeToContent="WidthAndHeight">
```

Comment:
It's good that it is filling the space, but it's filling it a little too much. It's putting the window menu bar above the top of my screen, and the window status bar below the bottom of my screen. On dual monitors, it does the same thing, so stretches to fill both monitors plus a little more. If you could fix this, that would be great.

Reply:
Agreed, I'm considering just maximizing the main editor window when it opens unless you have some objections?

Reply:
Yeah, let's give that a try.